lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -31,6 +52,10 @@
                    initialize: function (options) {
                        this.router = options.router || null;
                        this.tabs = options.tabs;
+                       // Convert each view into a TabPanelView
+                       _.each(this.tabs, function (tabInfo) {
+                           tabInfo.tabPanelView = new TabPanelView({tabInfo: tabInfo});
```

Comment:
Aha! I had thought of this, attempted it, and ended up with the circular reference as you described. I hadn't thought of `_.clone`, so I'll give that a shot.